msgraphcore/client_factory.py

```diff
@@ -27,15 +41,24 @@ def __init__(self, **kwargs):
         self._set_default_timeout()
 
     def create_with_default_middleware(self, credential: TokenCredential, **kwargs) -> Session:
-        """Applies the default middleware chain to the HTTP Client"""
+        """Applies the default middleware chain to the HTTP Client
+
+        :param credential: Accesstoken used to access the Graph API. Created through one of the
```

Comment:
`:param credential: Accesstoken used to access the Graph API.`

should be

`:param credential: TokenCredential used to acquire an access token for the Microsoft Graph API.`